TpmTestingPkg/TpmReplayPei/Tool/TpmReplay.py

```diff
@@ -37,8 +37,21 @@
     VALUE_FROM_EVENT,
     ALG_FROM_VALUE,
     EVENT_FROM_VALUE,
+    EV_EFI_VARIABLE_DRIVER_CONFIG,
+    EV_EFI_VARIABLE_BOOT,
+    EV_EFI_VARIABLE_BOOT2,
+    EV_EFI_VARIABLE_AUTHORITY
 )
 
+from edk2toollib.uefi.authenticated_variables_structure_support import EfiSignatureDatabase, EfiSignatureDataFactory, EfiSignatureDataEfiCertX509, EfiSignatureDataEfiCertSha256
+
+from edk2toollib.utility_functions import hexdump

```

Comment:
Can you please group all of the `edk2toollib` imports together (these two with the pre-existing `from edk2toollib.tpm.tpm2_defs import TPM_ALG_SHA256` one)?

Reply:
So I wasnt sure how to combine the two imports but I was able to clean them up. Do you have a suggestion?